projects/elife.yaml

```diff
@@ -1016,14 +999,6 @@ elife-api:
             1233: 80
 
 
-elife-reporting:
```

Comment:
that should be fine, there is no dependency on the project definition itself. And soon, hopefully, we might be able to detach the ejp scraping from lax (cc @de-code). The ejp scraping has been used to pre-populate lax with article 'stubs' and some things like submission and acceptance dates, that are in turn queried by Paul Kelly and exported to his dashboard. I think there are also RSS feeds that a pre-production service is using from lax that is using ejp data. I might make a 'containerise lax' epic-scoped task and fill it with these issues.

Reply:
actually, I'm reverting elife-reporting. I thought you were removing it because it had aws configuration. elife-reporting is a working environment of what production/James use to generate reports. It can be removed when James' reporting tools are no longer being used